src/server/status/routes/api/register_stats.js

```diff
@@ -0,0 +1,52 @@
+import Joi from 'joi';
+import { wrapAuthConfig } from '../../wrap_auth_config';
+
+/*
+ * API for Kibana meta info and accumulated operations stats
+ * Including ?extended in the query string fetches Elasticsearch cluster_uuid
+ * - Requests to set isExtended = true
+ *      GET /api/stats?extended=true
+ *      GET /api/stats?extended
+ * - No value or 'false' is isExtended = false
+ * - Any other value causes a statusCode 400 response (Bad Request)
+ */
+export function registerStatsApi(kbnServer, server, config, collector) {
+  const wrapAuth = wrapAuthConfig(config.get('status.allowAnonymous'));
+  server.route(
+    wrapAuth({
+      method: 'GET',
+      path: '/api/stats',
+      config: {
+        validate: {
+          query: {
+            extended: Joi.string().valid('', 'true', 'false')
+          }
+        },
+        tags: ['api'],
+      },
+      async handler(req, reply) {
+        const { extended } = req.query;
+        const isExtended = extended !== undefined && extended !== 'false';
+
+        let clusterUuid;
+        if (isExtended) {
+          try {
+            const { callWithRequest, } = server.plugins.elasticsearch.getCluster('data');
+            const { cluster_uuid: uuid } = await callWithRequest(req, 'info', { filterPath: 'cluster_uuid', });
+            clusterUuid = uuid;
+          } catch (err) {
+            clusterUuid = undefined; // fallback from anonymous access or auth failure, redundant for explicitness
```

Comment:
You don't really need to remove it, but it's implicitly `undefined` regardless of this line if we are catching an error.

Reply:
yeah, I'm leaving it to be redundant for explicitness, and also because it would fail lint to have an empty catch block

Reply:
I'm pretty sure that the linter will succeed given a comment, but this works too.